_delphi_utils_python/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY = venv, lint, test, clean
+
+dir = $(shell find ./delphi_* -name __init__.py |  sed 's/\/__init__.py//g;s/\.\///g')
+
+venv:
+	python3 -m venv env
+
+install: venv
+	. env/bin/activate; \
+	pip install wheel ; \
+	pip install -e .
+
+lint:
+	. env/bin/activate; \
+	pylint $(dir)
+
+test:
+	. env/bin/activate ;\
+	(cd tests && ../env/bin/pytest --cov=$(dir) --cov-report=term-missing)
+
+clean:
```

Comment:
Should `clean` also dump local `params.json` files?

Reply:
that's a good idea, though they are gitignored already. is there a possibility someone keeps them around for whatever reason?

Reply:
Scenario is this:

1. User runs `make clean install` to reset their local environment in response to major changes to indicator X
2. Tests fail and/or indicator can't run. `params.json.template` was updated, but local copy of `params.json` persisted after `clean`.

Reply:
makes sense, added.